.expeditor/promote-docker-images.sh

```diff
@@ -0,0 +1,42 @@
+#! /bin/bash
+
+export DOCKER_CLI_EXPERIMENTAL=enabled
+
+VERSION=$(cat VERSION)
```

Comment:
I don't think you want the contents of the VERSION file. Those contents could be different from the version that is being promoted. I think you can just have this script use `EXPEDITOR_VERSION`.

Reply:
Ah, yea - the whole head of promotion thing would make this a problem - good catch